frontends/election-manager/src/screens/tally_screen.tsx

```diff
@@ -230,199 +117,145 @@ export function TallyScreen(): JSX.Element {
-                  </TD>
-                </tr>
-                {castVoteRecordFileList.map(
-                  ({
-                    name,
-                    exportTimestamp,
-                    importedCvrCount,
-                    scannerIds,
-                    precinctIds,
-                  }) => (
-                    <tr key={name}>
-                      <TD narrow nowrap>
-                        {moment(exportTimestamp).format(
-                          'MM/DD/YYYY hh:mm:ss A'
-                        )}
-                      </TD>
-                      <TD nowrap>{format.count(importedCvrCount)} </TD>
-                      <TD narrow nowrap>
-                        {scannerIds.join(', ')}
-                      </TD>
-                      <TD>{getPrecinctNames(precinctIds)}</TD>
-                    </tr>
-                  )
-                )}
-                {externalTallyRows}
-                <tr>
-                  <TD as="th" narrow nowrap>
-                    Total CVRs Count
-                  </TD>
-                  <TD as="th" narrow>
-                    {format.count(
-                      castVoteRecordFileList.reduce(
-                        (prev, curr) => prev + curr.importedCvrCount,
-                        0
-                      ) + externalFileBallotCount
-                    )}
-                  </TD>
-                  <TD />
-                  <TD as="th" />
-                </tr>
-              </React.Fragment>
-            ) : (
-              <tr>
-                <TD colSpan={2}>
-                  <em>No CVR files loaded.</em>
-                </TD>
-              </tr>
-            )}
-          </tbody>
-        </Table>
-        <p>
-          <Button
-            onPress={() => setIsImportCvrModalOpen(true)}
-            disabled={isOfficialResults}
-          >
-            Import CVR Files
-          </Button>{' '}
-          {converter === 'ms-sems' && (
-            <React.Fragment>
-              <FileInputButton
-                innerRef={externalFileInput}
-                onChange={importExternalSemsFile}
-                accept="*"
-                data-testid="import-sems-button"
-                disabled={hasExternalSemsFile || isOfficialResults}
-              >
-                Import External Results File
-              </FileInputButton>{' '}
-            </React.Fragment>
-          )}
-          <LinkButton
-            to={routerPaths.manualDataImport}
-            disabled={isOfficialResults}
-          >
-            {hasExternalManualData
-              ? 'Edit Manually Entered Results'
-              : 'Add Manually Entered Results'}
-          </LinkButton>{' '}
-          <Button
-            disabled={!hasCastVoteRecordFiles || isOfficialResults}
-            onPress={confirmOfficial}
-          >
-            Mark Tally Results as Official
-          </Button>
-        </p>
-        {isOfficialResults ? (
-          <p>
+        <Prose maxWidth={false}>
```

Comment:
Yay! So glad we added the `Prose` wrapper here - it always struck me as unintentional that the text size and spacing on the tally tab was different than other tabs